src/modules/ttlmonitor/TTLMonitorInstrument.cpp

```diff
@@ -21,54 +22,49 @@ namespace TTLMonitor {
 
 /// \brief load configuration and calibration files
 TTLMonitorInstrument::TTLMonitorInstrument(
-    struct Counters &counters, BaseSettings &settings,
-    std::vector<EV44Serializer> &serializers)
+    struct Counters &counters, BaseSettings &settings)
 
-    : counters(counters), Settings(settings), Serializers(serializers) {
+    : counters(counters), Settings(settings) {
 
   XTRACE(INIT, ALW, "Loading configuration file %s",
          Settings.ConfigFile.c_str());
   Conf = Config(Settings.ConfigFile);
   Conf.loadAndApply();
 
-  if (!Settings.DumpFilePrefix.empty()) {
-    std::string DumpFileName =
-        Settings.DumpFilePrefix + "ttlmon_" + timeString();
-    XTRACE(INIT, ALW, "Creating HDF5 dumpfile: %s", DumpFileName.c_str());
-    DumpFile = VMM3::ReadoutFile::create(DumpFileName);
-  }
+  // if (!Settings.DumpFilePrefix.empty()) {
+  //   std::string DumpFileName =
+  //       Settings.DumpFilePrefix + "ttlmon_" + timeString();
+  //   XTRACE(INIT, ALW, "Creating HDF5 dumpfile: %s", DumpFileName.c_str());
+  //   DumpFile = VMM3::ReadoutFile::create(DumpFileName);
+  // }
```

Comment:
Should this be removed?

Reply:
Not, sure I am still considering to reinstate this, but there may be no need. We can delete it later.